package/system/apk/Makefile

```diff
@@ -64,7 +64,12 @@ MESON_ARGS += \
 	-Ddocs=disabled \
 	-Durl_backend=wget \
 	-Dcrypto_backend=$(BUILD_VARIANT) \
-	-Dzstd=false
+	-Dzstd=false \
+	-Dapplet-audit=false \
```

Comment:
Imho `apk audit` is a key feature and should not be missing on target systems by default.